index.js

```diff
@@ -2,66 +2,66 @@
   const canvas = document.getElementById("canvas");
   const context = canvas.getContext('2d');
   let gameSpeed = 30;
-  let computerPaddleSpeed = 5;
-  const MAX_SCORE = 3;
-  let playerScore = 0;
-  let computerScore = 0;
-
+  let paddleSpeed = 5;
 
+  class ObjXY {
+    constructor(x, y) {
+      this.x = x;
+      this.y = y;
+    }
+  }
 
   /**
-   * Ball Constructor
+   * Create instance of a ball
    */
-  function Ball() {
-    this.color = "white";
-    this.size = {
-      x: 10,
-      y: 10
+  class Ball {
+    constructor() {
+      this.color = "white";
+      this.size = new ObjXY(10, 10);
+      this.position = new ObjXY(
+        (canvas.width / 2) - (this.size.x / 2),
+        (canvas.height / 2) - (this.size.y / 2)
+      );
+      this.velocityX = -10;
+      this.velocityY = 10;
     };
-    this.position = {
-      x: (canvas.width / 2) - (this.size.x / 2),
-      y: (canvas.height / 2) - (this.size.y / 2)
-    };
-    this.velocityX = -10;
-    this.velocityY = 10;
-  };
 
-  Ball.prototype.calculatePosition = function () {
-    this.position.x = this.position.x + this.velocityX;
-    this.position.y = this.position.y + this.velocityY;
-  };
+    calculatePosition = () => {
```

Comment:
I think the method name should reflect that it is the next position being calculated, not its current position for example..
```suggestion
    calculateNextPosition = () => {
```